code/modules/mob/living/silicon/robot/drone/drone.dm

```diff
@@ -327,21 +327,6 @@
 	to_chat(src, "Remember,  you are <b>lawed against interference with the crew</b>. Also remember, <b>you DO NOT take orders from the AI.</b>")
 	to_chat(src, "Use <b>say ;Hello</b> to talk to other drones and <b>say Hello</b> to speak silently to your nearby fellows.")
 
-/mob/living/silicon/robot/drone/start_pulling(var/atom/movable/AM)
```

Comment:
Why did you remove this entire proc? All you had to do is remove single check from it.

Reply:
I took it out because all the checks it was doing were already checked inside of mob.dm here https://github.com/Aurorastation/Aurora.3/blob/master/code/modules/mob/mob.dm#L656
I didn't realise though that it posts a slightly different message, I guess that's the purpose of it.

Reply:
Please put the proc back. it is cleaner this way, I tested it and it works. Here is the code for it.
```
/mob/living/silicon/robot/drone/start_pulling(var/atom/movable/AM)
	if(!AM) return
	for(var/A in pull_list)
		if(istype(AM, A))
			if(pulling)
				var/pulling_old = pulling
				stop_pulling()
				// Are we pulling the same thing twice? Just stop pulling.
				if(pulling_old == AM)
					return

			src.pulling = AM
			AM.pulledby = src

			if(pullin)
				pullin.icon_state = "pull1"
				return
			return
	if(istype(AM, /obj/item))
		var/obj/item/O = AM
		if(O.w_class > can_pull_size)
			to_chat(src, "<span class='warning'>You are too small to pull that.</span>")
			return
	else
		if(!can_pull_mobs)
			to_chat(src, "<span class='warning'>You are too small to pull that.</span>")
			return
	..()
```

Reply:
Also add `var/list/pull_list = list(/obj/structure/disposalconstruct)` after line 60

Reply:
Ah I see, excellent thanks for your help :D